docs/src/components/footer/slim.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+
+export const SlimFooter = () => {
+	return (
+		<footer className="usa-footer usa-footer--slim">
+			<div className="grid-container usa-footer__return-to-top">
+				<Link to="#">Return to top</Link>
```

Comment:
It should be safe to put #top as the href for all of these.  Bryan and I did some digging and all modern browsers understand it.